Core/src/main/java/org/tribuo/transform/TransformTrainer.java

```diff
@@ -29,6 +29,9 @@
 
 import java.time.OffsetDateTime;
 import java.util.Map;
+import java.util.logging.Handler;
```

Comment:
Not sure we need the Handler import once the debugging statements have been removed.